### PR TITLE
feat: introduced endpoint to get vaccines for the patient

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/dispensary/views.py [hunk 1]
```diff
@@ -21,6 +21,12 @@ def list(self, request):
         serializer = MedicineSerializer(queryset, many=True)
         return Response(serializer.data)
 
+     @action(detail=False, methods=['get'], url_path='vaccines')
+     def get_vaccines(self, request):
+        queryset = Medicine.objects.filter(description="Vaccine")
```
Comment (Raam124) [inline apps/dispensary/views.py:26]: u cant add filters by a string value maintain a medicine type in this case

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/views.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/dispensary/views.py]: added choice field in the models filtered to medicine type

**Review by Raam124 — APPROVED** [review]

File: apps/dispensary/views.py [hunk 2]
```diff
@@ -21,6 +21,12 @@ def list(self, request):
         serializer = MedicineSerializer(queryset, many=True)
         return Response(serializer.data)
 
+     @action(detail=False, methods=['get'], url_path='vaccines')
+     def get_vaccines(self, request):
+        queryset = Medicine.objects.filter(medicine_type='VACCINE')
```
Comment (Raam124) [inline apps/dispensary/views.py:26]: filter by type instead of string here

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/dispensary/views.py]: Apologies, added filtration for type

**Review by Raam124 — APPROVED** [review]

**Pull request merged**